apps/console/src/features/identity-providers/components/wizards/oidc-authentication-wizard-page.tsx

```diff
@@ -150,7 +150,7 @@ export const OidcAuthenticationWizardFrom = (props: OidcAuthenticationWizardFrom
                         name="authorizationEndpointUrl"
                         label={ "Authorization endpoint URL" }
                         required={ true }
-                        placeholder={ "https://myorg.io/authorize/" }
+                        placeholder={ "https://ENTERPRISE_DOMAIN/authorize" }
```

Comment:
Should this be `ENTERPRISE_DOMAIN` or `<ENTERPRISE_DOMAIN>
@minolih

Reply:
Verified this. We are going ahead with https://ENTERPRISE_DOMAIN/authorize as a placeholder inside a placeholder may confuse users.